README.md

```
@@ -1,128 +1,125 @@
+  * [🔗 디자인 산출물 바로가기 (figma)](https://www.figma.com/design/bO3fqDgEZfQ3oCGJTHlM5d/Navisa-Handoff?node-id=170-53&t=8YyjgisB0pJL51Zg-1)
+
   
+<br>
+
+# 🛠️ 기술적 도전 및 해결
+
+> **Navisa팀은 단순한 기능 구현을 넘어, 데이터의 신뢰성과 시스템의 안정성을 확보하기 위해 끊임없이 고민했습니다.**
+> 
+> 자세한 내용은 팀 WIKI에서 확인하실 수 있습니다.
+
+### 1. 고도화된 매칭 엔진 설계
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)